doc/commands.md

```diff
@@ -220,7 +220,7 @@ Option | Description | Type | Notes
 `--maxChunks` | Limit the number of Webpack chunks in build output | number |
 `--minify` | Minify the bundle output | boolean | default: true
 `--okapi` | Specify an Okapi URL | string |
-`--output` | Directory to place build output | string |
+`--output` | Directory to place build output (if omitted, default value `./output` is used) | string |
```

Comment:
Did you update manually?  FYI, `yarn docs` should ensure all commands are updated and stamp the version at the top.

Reply:
I did update manually. I can revert to let the automatic process take over.

Reply:
No need to revert.  You can just run the docs script now or anytime and it will automatically sync commands.md.  Just an FYI since the next time its run, your text will be overwritten.